yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/SecureYotiClient.java

```diff
@@ -159,48 +161,35 @@ private EncryptedData parseProfileContent(byte[] profileContent) throws ProfileE
         }
     }
 
-    private Key parseKey(byte[] keyVal) {
-        return new SecretKeySpec(keyVal, SYMMETRIC_CIPHER);
-    }
-
-    private List<Attribute> parseProfile(byte[] profileData) throws ProfileException {
+    private List<Attribute<?>> parseProfile(byte[] profileData) throws ProfileException {
         try {
             AttributeList message = AttributeList.parseFrom(profileData);
-            List<Attribute> attributeList = parseAttributes(message);
+            List<Attribute<?>> attributeList = parseAttributes(message);
             LOG.debug("{} attribute(s) parsed", attributeList.size());
             return attributeList;
         } catch (InvalidProtocolBufferException e) {
             throw new ProfileException("Cannot parse profile data", e);
         }
     }
 
-    private List<Attribute> parseAttributes(AttributeList message) {
-        List<Attribute> parsedAttributes = new ArrayList<>();
+    private List<Attribute<?>> parseAttributes(AttributeList message) {
+        List<Attribute<?>> parsedAttributes = new ArrayList<>();
         for (AttrProto.Attribute attribute : message.getAttributesList()) {
             try {
-                parsedAttributes.add(AttributeConverter.convertAttribute(attribute));
-            } catch (IOException e) {
-                LOG.info("Cannot decode value for attribute {}", attribute.getName());
-            } catch (ParseException e) {
-                LOG.info("Cannot parse value for attribute {}", attribute.getName());
+                parsedAttributes.add(attributeConverter.convertAttribute(attribute));
+            } catch (IOException | ParseException e) {
+                LOG.warn("Cannot parse value for attribute '{}'", attribute.getName());
```

Comment:
I know that is like this before but I'd like to keep the exception in the logs to see what could be the cause, do you agree?

Reply:
I've replied in the SDK-385 ticket rather than here in the public repo...